src/convertToObject.js

```diff
@@ -6,7 +6,29 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  // First - clear string from other symbols
+  const tempString = sourceString.trim().replace(/;+\s*$/, '');
+  const cssList = tempString.split(';');
+  const cssObject = {};
+
+  cssList.forEach((style) => {
+    let [key, value] = style.split(':');
+
+    if (key && value) {
+      key = key.trim();
+      value = value.trim();
```

Comment:
You should also consider handling cases where the style declaration might contain a colon in the value part (e.g., a background-image with a data URI). Currently, the 'split' operation would incorrectly handle such cases.